rest_framework/authtoken/serializers.py

```diff
@@ -18,13 +19,25 @@ def validate(self, attrs):
             if user:
                 if not user.is_active:
                     msg = _('User account is disabled.')
-                    raise serializers.ValidationError(msg)
+                    raise serializers.ValidationError(
+                        ValidationErrorMessage(
+                            msg,
+                            code='authorization')
```

Comment:
> Just to clarify, are you against the fact that we put the code on the string-like object?
> Because right now I don't quite see how we could maintain backward compatibility without that.

Well there's two issues:

1. Where it goes in the `ValidationError` - an optional `code` argument here would be fine.
2. How we present it on the serializer - seeing what Django does in the Forms API would be useful here.